packages/graphql/lib/src/websocket/messages.dart

```diff
@@ -74,15 +74,15 @@ class LegacyInitOperation extends InitOperation {
   LegacyInitOperation(dynamic payload) : super(payload);
 
   @override
-  dynamic toJson() {
+  Map<String, dynamic> toJson() {
     final Map<String, dynamic> jsonMap = <String, dynamic>{};
     jsonMap['type'] = type;
 
     if (payload != null) {
       jsonMap['payload'] = json.encode(payload);
     }
 
-    return json.encode(jsonMap);
+    return jsonMap;
```

Comment:
Why are we removing JSON encoder?

Reply:
as explain the description of this PR, toJson() should always return a <String, dynamic> not something else, especially not String.